packages/kotti-ui/source/kotti-button/KtButton.vue

```diff
@@ -1,23 +1,20 @@
 <template>
-	<div class="kt-button-wrapper" style="display: contents">
-		<button
-			ref="triggerRef"
-			v-bind="$attrs"
-			:class="mainClasses"
-			role="button"
-			:type="isSubmit ? 'submit' : 'button'"
-			@click="handleClick"
-		>
-			<i v-if="isLoading" class="kt-circle-loading" />
-			<i v-if="hasIconLeft" class="yoco" v-text="icon" />
-			<span v-if="hasSlot">
-				<slot />
-			</span>
-			<span v-else-if="label !== null" v-text="label" />
-			<i v-if="hasIconRight" class="yoco" v-text="icon" />
-		</button>
-		<div v-if="showHelpText" ref="contentRef" v-text="helpText" />
-	</div>
+	<button
+		ref="helpTextTriggerRef"
+		:class="mainClasses"
+		role="button"
```

Comment:
why do we need a role button on a button

It is not needed because it is the default value for a button anyways

see https://www.w3.org/TR/2014/REC-html5-20141028/dom.html#aria-role-attribute

or look for this note if the anchor link doesn't work

```
In the majority of cases setting an ARIA [role](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#attr-aria-role) and/or aria-* attribute that matches the [default implicit ARIA semantics](https://www.w3.org/TR/2014/REC-html5-20141028/infrastructure.html#default-implicit-aria-semantics) is unnecessary and not recommended as these properties are already set by the browser
```

Reply:
ahh! good catch